grpc/pom.xml

```diff
@@ -50,6 +50,11 @@
       <version>${grpc.version}</version>
     </dependency>
     <!-- Other -->
+    <dependency>
+      <groupId>com.datastax.oss</groupId>
+      <artifactId>java-driver-query-builder</artifactId>
+      <version>4.13.0</version>
+    </dependency>
```

Comment:
I don't like this. That query builder is tied to the driver, in fact it pulls a dependency to it.
I'm going to try to use our internal query builder instead.